Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Mon, 5 Apr 2021 18:12:06 -0400
+Subject: [PATCH] Add EntityBlockStorage#clearEntities()
+
+
+diff --git a/src/main/java/org/bukkit/block/EntityBlockStorage.java b/src/main/java/org/bukkit/block/EntityBlockStorage.java
+index f3f8d765d5d487aa14edf0b88c32608fb804c331..eb48b7e8275080bea08c641406aaeecad7093735 100644
+--- a/src/main/java/org/bukkit/block/EntityBlockStorage.java
++++ b/src/main/java/org/bukkit/block/EntityBlockStorage.java
+@@ -53,4 +53,10 @@ public interface EntityBlockStorage<T extends Entity> extends TileState {
+      * @param entity Entity to add to the block
+      */
+     void addEntity(@NotNull T entity);
++    // Paper start - Add EntityBlockStorage clearEntities
```

Comment:
New line

Reply:
What do you mean newline here? Before or after the paper start?

Reply:
Before